components-dist/components/shared/Buttons/CollapisbleButton.d.ts

```
@@ -0,0 +1,2 @@
+declare const CollapsibleButton: ({ children, ...restProps }: any) => import("react/jsx-runtime").JSX.Element;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)